src/tile/tile.scss

```diff
@@ -313,3 +315,30 @@
     @include visually-hidden;
   }
 }
+
+@mixin iui-tile-array {
+  width: 100%;
+
+  // #region IE support
```

Comment:
I'm wondering two things:
- does this need to go in this repo?
- does this need to support IE?

Reply:
IE support I can take it or leave it, I added it because it was pretty light weight & it didn't take much time.

Regarding where this belongs, I think it belongs in CSS rather than layouts.  If a team wanted to use our responsive tiles and nothing else this way they'd only need to import 1 pkg, not 2.

Reply:
IE support is removed.